src/frontend/components/DashboardVideoLiveControlPane/inputs/DashboardVideoLiveWidgetToggleInput/index.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DashboardVideoLiveWidgetToggleInput } from './index';
+
+const onChangeToggleMock = jest.fn();
+
+describe('<DashboardVideoLiveWidgetToggleInput />', () => {
+  beforeEach(() => jest.resetAllMocks());
+  it('renders the toggle input unchecked and performs a click on it', () => {
+    render(
+      <DashboardVideoLiveWidgetToggleInput
+        checked={false}
+        label="An example title"
+        onChange={onChangeToggleMock}
+      />,
+    );
+
+    screen.getByText('An example title');
+
+    const toggleInput = screen.getByRole('checkbox', {
+      name: 'An example title',
+    });
+    userEvent.click(toggleInput);
+
+    expect(onChangeToggleMock).toHaveBeenCalledTimes(1);
```

Comment:
You can also check the toggle state itself. Should it be checked or not ?

Reply:
I added tests to check toggle are checked or unchecked depending on what's given in the props. But I can't test toggle's state is changing after performing a click on it, because it is not supposed to change (indeed, the state of the toggle can only by changing by what is given in props).

Reply:
Sorry, `check the toggle state` means testing if the checkbox is checked or not. It was not about the internal component state.

Reply:
ok, so I think its done in [1820cacf1e](https://github.com/openfun/marsha/pull/1503/commits/1820cacf1eed8ffc0e18508f83db9e66a169fba9)